### ISSUE TITLE
Manually setting color sets using JSON-based config not working

### ISSUE THREAD

**Issue by davidliang2008** [amcharts/amcharts4#577]
Based the JSON-based config tutorial, it looks like you can set a hard-coded list of colors globally for the chart: https://www.amcharts.com/docs/v4/concepts/json-config/#Structure_of_JSON_config

```javascript
{
  "colors": {
    "list": ["#ff0000", "#00ff00", "#0000ff" ]
  }
}
```

There is also a documentation stating you can assign colors to the series: https://www.amcharts.com/docs/v4/concepts/colors/#Manually_setting_color_sets

```javascript
{
  // ...
  "series": [{
    // ...
    "colors": [{
      "#845EC2",
      "#D65DB1",
      "#FF6F91",
      "#FF9671",
      "#FFC75F",
      "#F9F871"
    }]
  }]
}
```

Both cases don't override the default color set.

**demo:** http://jsfiddle.net/aq9Laaew/274847/

**Comment by martynasma** [comment]
Thanks for the heads up.

Looks like there are two issues: documentation is wrong and there's a bug that prevents it from working :)

I updated the docs. It should say this:

```
{
  // ...
  "series": [{
    // ...
    "colors": {
      "list": [
        "#845EC2",
        "#D65DB1",
        "#FF6F91",
        "#FF9671",
        "#FFC75F",
        "#F9F871"
      ]
    }
  }]
}
```

Also, fixed bug in dev version. New version will be released within 1-2 days.

Will ping you once that happens.

**Comment by martynasma** [comment]
Fixed in [4.0.0-beta.85](https://github.com/amcharts/amcharts4/releases/tag/4.0.0-beta.85).

## [4.0.0-beta.85] - 2018-11-20

### Added
- `"ready"` event added to `Sprite`. Sprite fires it right after `"init"` event. `Container` fires it when all of its children are ready.
- `system.isPaused` property added. Setting it to `true` will effectively pause any chart activity, making them completely static.
- `ErrorBullet` class added. Allows drawing error charts. (Check `error-chart` and `xy-error-chart` examples for code samples)
- JSON: String-based `cursorOverStyle` and `cursorDownStyle` are not supported.

### Fixed
- JSON: Setting `ColorSet` properties `baseColor` or `list`  was not working properly.
- Accessibility: Legend item was not properly referring to a Series via `labelled-by`.
- `Label.wrap` was breaking lines mid-word even if there where words that would fit into alotted space.
- Memory leak fixed.
- Sometimes `Animation` was not firing `"animationended"` event (when elastic easing was used).

[Full change log](https://github.com/amcharts/amcharts4/blob/master/dist/es2015/CHANGELOG.md).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
